### DISCUSSION TITLE
Output inteval flag seems not work if adaptivity mesh is turned on

### DISCUSSION THREAD

**Discussion by ZhigangPu** [idaholab/moose#18670] [Q&A General] [answered]
Hi all, I am new to MOOSE and I have trouble dealing with output when I turn on the adaptivity mesh refining. My disk capacity is limited so I don't want MOOSE to output results every timestep. It's fine when I don't use adaptivity mesh refining(AMR), there is only one exodus output file and 'inteval' flag works. But when I turn on AMR, MOOSE will output a exodus file every step, whose suffix is '.e-sxxx'(xxx is timestep number). I wonder if there is some flags that can turn off this. For you better reference, below is my configuration for AMR and output.

Thanks in advance!

>  [./Adaptivity]
>     initial_adaptivity = 6 # Number of times mesh is adapted to initial condition
>     refine_fraction = 0.7 # Fraction of high error that will be refined
>     coarsen_fraction = 0.1 # Fraction of low error that will coarsened
>     max_h_level = 7 # Max number of refinements used, starting from initial mesh (before uniform refinement)
>     weight_names = 'c   eta'
>     weight_values = '1  1'
>     inteval = 5
>   [../]
> 
> [Outputs]
> [./exodus]
>   type = Exodus
>   inteval = 3
>   execute_on = 'TIMESTEP_END'
> [../]
> [./my_checkpoint]
>   type=Checkpoint
>   num_files = 4
>   interval = 30
> [../]
> []

**Comment by GiudGiud** [comment, marked as answer]
Hello

`interval` is misspelled in the Exodus block. Could that be the problem?

Guillaume

**Reply by ZhigangPu** [reply]
Hello Guillaume
Thanks for pointing out the typo. I've changed the typo and re-run, but things are the same.
Thanks anyway!

zhigang

**Reply by GiudGiud** [reply]
did you catch the typo in the [Adaptivity] block as well? Not sure if you were trying to limit adaptivity to limit output here
If `interval` does not work for your case, I dont see any other parameters that would reduce the number outputs in a similar way.
I'd have to look in the code to see why `interval` doesnt work

**Reply by ZhigangPu** [reply]
Oh, I think I get the point.
I thought the suffix of the output exodus file stands for the step. For example, 'xxx.e-s002' stands for time step 002, but actually it is not so. It only means that this is a second output file, the real time step it represents may be 3, 10, 20, whatever depending on my `interval` setting. But to be honest, I think it is a little bit misleading. I think I will try to find ways to change the naming rule.
And I don't need to add `inteval` flags in the [Adaptivity] block, `inteval` in [Output] block is enough.
Thanks!

zhigang
